packages/azure-themes/src/azure/AzureThemeDark.ts

```diff
@@ -8,7 +8,7 @@ const darkExtendedSemanticColors: Partial<IExtendedSemanticColors> = {
   commandBarBorder: DarkSemanticColors.commandBar.border,
   bodyBackground: DarkSemanticColors.background,
   bodyText: DarkSemanticColors.text.body,
-  bodyTextHovered: DarkSemanticColors.text.bodyHovered,
+  bodyTextHover: DarkSemanticColors.text.bodyHovered,
```

Comment:
Sorry, can you change it back to bodyTextHovered? 

I looked around the codebase and it looks like the following format is used for hover variable naming:
- xxxHovered
- xxxOnHover

So either bodyTextHovered or bodyTextOnHover would work. For this case, we can keep the original name.

Reply:
Got it, thank you

Reply:
Complete